sea-orm-macros/src/derives/active_enum.rs

```diff
@@ -337,6 +337,16 @@ impl ActiveEnum {
                 }
             }
 
+            #[automatically_derived]
+            impl sea_orm::TryGetableArray for #ident {
+                fn try_get_by<I: sea_orm::ColIdx>(res: &sea_orm::QueryResult, index: I) -> std::result::Result<Vec<Self>, sea_orm::TryGetError> {
+                    <<Self as sea_orm::ActiveEnum>::Value as sea_orm::ActiveEnumValue>::try_get_vec_by(res, index)?
+                        .into_iter()
+                        .map(|value| Self::try_from_value(&value).map_err(Into::into))
```

Comment:
@tyt2y3 This needs to be changed from `Self::try_from_value(&value)` to `<Self as sea_orm::ActiveEnum>::try_from_value(&value)`

Reply:
I see!